src/libgap-api.c

```diff
@@ -95,6 +95,15 @@ Obj GAP_ValueGlobalVariable(const char * name)
     }
 }
 
+int GAP_AssignGlobalVariable(const char * name, Obj value)
+{
+    UInt gvar = GVarName(name);
+    if (IsReadOnlyGVar(gvar)) {
```

Comment:
super-picky, you might want IsReadOnlyGVar(gvar) || IsConstantGVar(gvar)

Reply:
Good point. Fixed.